src/Aquifer.Public.API/Endpoints/Bibles/Alignments/Greek/Endpoint.cs

```diff
@@ -0,0 +1,228 @@
+        LEFT JOIN GreekNewTestamentWords gntw ON gntw.Id = gntwgw.GreekNewTestamentWordId
+        LEFT JOIN GreekNewTestaments gnt ON gntw.GreekNewTestamentId = gnt.Id
+        WHERE BibleId = @BibleId
+        """;
+
+    private const string BaseQuery = """
+        SELECT bvw.Id, bvw.WordIdentifier, bvw.[Text] AS EnglishWord, bvwgw.BibleVersionWordGroupId
+        FROM BibleVersionWords bvw
+        LEFT JOIN BibleVersionWordGroupWords bvwgw ON bvwgw.BibleVersionWordId = bvw.Id
+        WHERE BibleId = @BibleId AND bvw.WordIdentifier BETWEEN @LowerBounds AND @UpperBounds
+        ORDER BY bvw.WordIdentifier
+        """;
+
+    private const string GreekWordsQuery = """
+        SELECT bvw.Id AS EnglishWordId, gntw.Id AS GreekNewTestamentWordId, gw.[Text] AS GreekWord
+        FROM BibleVersionWords bvw
+        LEFT JOIN BibleVersionWordGroupWords bvwgw ON bvwgw.BibleVersionWordId = bvw.Id
+        LEFT JOIN NewTestamentAlignments nta ON nta.BibleVersionWordGroupId = bvwgw.BibleVersionWordGroupId
+        LEFT JOIN GreekNewTestamentWordGroupWords gntwgw ON gntwgw.GreekNewTestamentWordGroupId = nta.GreekNewTestamentWordGroupId
+        LEFT JOIN GreekNewTestamentWords gntw ON gntw.Id = gntwgw.GreekNewTestamentWordId
+        LEFT JOIN GreekWords gw ON gntw.GreekWordId = gw.Id
+        WHERE BibleId = @BibleId AND bvw.WordIdentifier BETWEEN @LowerBounds AND @UpperBounds AND gntw.Id IS NOT NULL
+        """;
+
+    private const string GreekSensesQuery = """
+        WITH CTE AS (
+            SELECT gntw.Id AS GreekNewTestamentWordId,
+                    gs.DefinitionShort AS Definition,
+                    gsg.[Text] AS Gloss,
+                    ROW_NUMBER() OVER (PARTITION BY gntw.Id, gs.DefinitionShort, gsg.[Text] ORDER BY gntw.Id) AS RowNum
+            FROM BibleVersionWords bvw
+            LEFT JOIN BibleVersionWordGroupWords bvwgw ON bvwgw.BibleVersionWordId = bvw.Id
+            LEFT JOIN NewTestamentAlignments nta ON nta.BibleVersionWordGroupId = bvwgw.BibleVersionWordGroupId
+            LEFT JOIN GreekNewTestamentWordGroupWords gntwgw ON gntwgw.GreekNewTestamentWordGroupId = nta.GreekNewTestamentWordGroupId
+            LEFT JOIN GreekNewTestamentWords gntw ON gntw.Id = gntwgw.GreekNewTestamentWordId
+            LEFT JOIN GreekNewTestamentWordSenses gntws ON gntws.GreekNewTestamentWordId = gntw.Id
+            LEFT JOIN GreekSenses gs ON gs.Id = gntws.GreekSenseId
+            LEFT JOIN GreekSenseGlosses gsg ON gsg.GreekSenseId = gs.Id
+            WHERE BibleId = @BibleId AND bvw.WordIdentifier BETWEEN @LowerBounds AND @UpperBounds AND gntw.Id IS NOT NULL AND gs.DefinitionShort IS NOT NULL
+        )
+        SELECT GreekNewTestamentWordId, Definition, STRING_AGG(Gloss, '||') AS Glosses
+        FROM CTE
+        WHERE RowNum = 1
+        GROUP BY GreekNewTestamentWordId, Definition
+        """;
```

Comment:
I'm having trouble with the performance of this query (see PR description).  This PR is blocked until I can address it.

Reply:
Really weird.  Removing one level of indentation (four spaces) under the SELECT fixed it.  I think that the whitespace must somehow impact the query plan chosen by SQL Server.